### PR TITLE
TW-915: Fix click on notification to open room in web

### PR DESCRIPTION
**Pull request by nqhhdev**
### Issue:

- Can't click on notifications on web

#### Chrome

https://github.com/linagora/twake-on-matrix/assets/99852347/624f126b-ba39-4bb5-9344-d80176dc34df

### CODE REVIEW

File: web/script.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            if (permission === 'granted') {
+                var notification = new Notification(title, {
+                    body: body,
+                    icon: icon,
+                });
+
+                notification.onclick = function() {
+                    console.log('JsFunction::handleNotifications(): On click notification');
+                    var host = window.location.host;
+                    var redirectURL = 'http://' + host + '/#/rooms/' + roomid;
```
Comment (sherlockvn) [inline web/script.js:13]: what about `https://`

File: lib/widgets/local_notifications_extension.dart [hunk 2]
```diff
@@ -10,13 +9,19 @@ import 'package:http/http.dart' as http;
...
+import 'package:universal_html/js.dart' as js;
 
 import 'package:fluffychat/config/app_config.dart';
 import 'package:fluffychat/utils/matrix_sdk_extensions/matrix_locals.dart';
 import 'package:fluffychat/utils/platform_infos.dart';
 import 'package:fluffychat/widgets/matrix.dart';
 
 extension LocalNotificationsExtension on MatrixState {
+  static final html.AudioElement _audioPlayer = html.AudioElement()
+    ..src = 'assets/assets/sounds/WoodenBeaver_stereo_message-new-instant.ogg'
```
Comment (sherlockvn) [inline lib/widgets/local_notifications_extension.dart:21]: why double `asset/asset` ?

File: web/script.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                var notification = new Notification(title, {
+                    body: body,
+                    icon: icon,
+                });
+
+                notification.onclick = function() {
+                    console.log('JsFunction::handleNotifications(): On click notification');
+                    var host = window.location.host;
+                    var protocol = window.location.protocol;
+                    var redirectURL = protocol + '//' + host + '/#/rooms/' + roomid;
```
Comment (drminh2807) [inline web/script.js:14]: do not work on github page and beta twake
https://linagora.github.io/twake-on-matrix/971
better is callback flutter channel

Comment (nqhhdev) [reply web/script.js:14]: In dart code we can't handle it. Only handle it when using Js code

**Review by hoangdat — APPROVED** [review]

**Review by sherlockvn — APPROVED** [review]

**Review by imGok — APPROVED** [review]

**Pull request merged**